AsyncDisplayKitTests/ASStackLayoutSpecSnapshotTests.mm

```diff
@@ -63,6 +65,24 @@ - (void)testStackLayoutSpecWithJustify:(ASStackLayoutJustifyContent)justify
   [self testStackLayoutSpecWithStyle:style sizeRange:sizeRange subnodes:subnodes identifier:identifier];
 }
 
+- (void)testStackLayoutSpecWithDirection:(ASStackLayoutDirection)direction
```

Comment:
As always, thank you for implementing tests!  It is extra effort — especially to create snapshot tests — but truly valuable to the community and sets a great example.